extensions/csharp/package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "vscode",
   "license": "MIT",
   "engines": {
-    "vscode": "0.10.x"
+    "vscode": "^0.10.0"
```

Comment:
We should avoid making updates to VS Code stuff unless necessary - please revert the changes to extensions coming from them

Reply:
They are all necessary changes that I've made so far as `@vscode/vsce` now scans extensions better and it's catching all potential issues.

Reply:
Yes, but unless it's a security concern (which for build stuff it shouldn't be) then for the sake of making merges easier it's preferable to wait until VS Code updates their stuff.

Reply:
Ok, downgrading to v2.16.0 for now.